From 4c990347794779425829aabd7af224597a0de2d0 Mon Sep 17 00:00:00 2001
From: Hauke Mehrtens <hauke.mehrtens@intel.com>
Date: Fri, 29 May 2020 18:30:05 +0200
Subject: [PATCH] PONRTSYS-6699: Update PPA QoS manager header files

This fixes a compile problem in the PON QoS Manager.
The PPA header files were moved in commit 924a86433cc1 ("UGW_SW-45485:
Segregation of QoS manager and acceleration logic")

Fixes: c8b9330b85a2 ("PONRTSYS-6699:")
---
 drivers/net/datapath/pon_qos/pon_qos_tc_main.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/drivers/net/datapath/pon_qos/pon_qos_tc_main.c b/drivers/net/datapath/pon_qos/pon_qos_tc_main.c
index 5e6ab49b1ff9..bafa90997bac 100644
--- a/drivers/net/datapath/pon_qos/pon_qos_tc_main.c
+++ b/drivers/net/datapath/pon_qos/pon_qos_tc_main.c
@@ -9,7 +9,7 @@
 #include "pon_qos_tc_main.h"
 #include "pon_qos_tc_qos.h"
 #if IS_ENABLED(CONFIG_QOS_MGR)
-#include <net/ppa/qos_mgr_tc_hook.h>
+#include <net/qos_mgr/qos_mgr_tc_hook.h>
 #endif
 
 #define DRV_NAME	"mod_pon_qos"
